README.md

```diff
@@ -1,46 +1,123 @@
 # Rails URL Helpers
 
-## Objectives
+Since Rails by design was meant to be flexible, the result is that there are typically a number of ways to accomplish the same features. Routes are a great example for how this principle operates in a Rails app. As you have already seen in our Capybara feature specs, we have the ability to hardcode paths or to use view helper methods. In this section we will extend this knowledge and walk though both options and how they can be used in the implementation code itself.
```

Comment:
Drop the "as you have already seen in Capybara..." because they won't have seen that yet because we yanked it out